### PR TITLE
Feature/bocm 53 integrate get appeals with prisma db

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

Adds integration with the database for "get appeals for validation" request

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/prisma/seed.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import Prisma from '@prisma/client';
```
Comment (radum) [inline apps/api/prisma/seed.js:1]: Formatted :)

```js
import Prisma from '@prisma/client';

const { PrismaClient } = Prisma;

const prisma = new PrismaClient();

const appealsData = [
	{
		reference: 'SOME REFERENCE',
		address: {
			create: {
				addressLine1: 'first line',
				city: 'city',
				postcode: 'A11 0ZZ'
			}
		}
	}
];

async function main() {
	console.log(`Start seeding ...`);
	for (const a of appealsData) {
		const appeal = await prisma.appeal.create({
			data: a
		});
		console.log(`Created appeal with id: ${appeal.id}`);
	}
	console.log(`Seeding finished.`);
}

main()
	.catch((e) => {
		console.error(e);
		process.exit(1);
	})
	.finally(async () => {
		await prisma.$disconnect();
	});

```

**Review by radum — DISMISSED** [review]

File: apps/api/prisma/seed.js (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/api/prisma/seed.js:1]: 👍

**Review by radum — APPROVED** [review]

**Pull request merged**